client/Lykke.Service.MarketMakerReports.Client/Models/Trades/TradeType.cs

```diff
@@ -0,0 +1,8 @@
+namespace Lykke.Service.MarketMakerReports.Client.Models.Trades
+{
+    public enum TradeType
```

Comment:
Add default value "None" to indicate unspecified type

Reply:
Why to add into the contract an invalid value? We don't have trades which are neither Sell no Buy at the first place.